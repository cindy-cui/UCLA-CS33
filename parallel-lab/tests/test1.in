0.1
9.8
1000
5
5 10 1000
10 5 5
100 14 4774
172 57 687
1 487 9990
