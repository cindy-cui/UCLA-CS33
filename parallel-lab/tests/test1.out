382.176	-16.0995
424.6	130.446
217.272	-54.9931
374.139	93.9343
23.2899	428.681
avg acceleration=(x=45.3391 m/s^2, y=14.7379 m/s^2)
avg displacement/second=(x=2.26695 m, y=0.736894 m)
